### PR TITLE
[Export CSV] Add WKT to exported CSV

### LINKED ISSUE #797: [Export CSV] Add geometries to export
@kdyson This will make the exported CSV much larger. Is including the whole geometry in each row the most useful format for your work, or would it be more useful to export an ID or other reference?

### PR DESCRIPTION
**Pull request by gino-m**
Fixes #797. @DaoyuT, PTAL?

### CODE REVIEW

**Comment by gino-m** [comment]
@DaoyuT Gentle ping

File: functions/export-csv.js [hunk 1]
```diff
@@ -101,6 +104,30 @@ async function exportCsv(req, res) {
...
 }
 
+function toWkt(geoJsonString) {
+  const geoJsonObject = parseGeoJson(geoJsonString);
+  const geometry = getGeometry(geoJsonObject);
+  return geometry ? wkt.stringify(geometry) : "";
+}
+
+function parseGeoJson(jsonString) {
+  if (!jsonString) {
```
Comment (DaoyuT) [inline functions/export-csv.js:114]: nit: JSON.parse(null) gives you null without error, so you can remove this if.

**Review by DaoyuT — DISMISSED** [review]
LGTM, just 1 nit comment.

File: functions/export-csv.js (hunk 1 above)
Comment (gino-m) [reply functions/export-csv.js]: Done.

**Comment by gino-m** [comment]
@DaoyuT Restamp please? :)

**Pull request merged**